frontend/src/pages/chartTest.js

```diff
@@ -0,0 +1,117 @@
+import React, { useEffect, useRef, useState } from 'react';
+import Chart from 'chart.js';
+import Button from 'react-bootstrap/Button';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import Container from 'react-bootstrap/Container';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+
+import Radar from './chart/chart';
+
+
+function RadarTest(props) {
+
+    const [input, setInput] = useState(0);
+    const [testInput1, settestInput1] = useState(0);
```

Comment:
is this the state for the 3 issues? I feel like it would make more sense to be along the lines of 
`useState([0,0,0])` rather than 3 state objects

Reply:
These states for used for the input values of the issues, so it stores whatever the input value is for each input text box. Is there way to use onChange and only alter one element of the array?